### PR TITLE
PVC storage upgrade

### PR DESCRIPTION
**Pull request by FreshGyan**
Switching gluster storage from file to block. The speed difference is significant.
Also switching database backups to the nfs-backup storage type PVC because pathfinder team backs it up constantly

Workflow:

- [ ] Get code review
- [ ] Backup data from TEST and PROD
- [ ] Merge code
- [ ] Wait till code reaches TEST
- [ ] Copy data over from old drives to new one
- [ ] Merge data to PROD
- [ ] Copy data over from old drives to new one

### CODE REVIEW

File: pipeline/config-prod.groovy [hunk 1]
```diff
@@ -273,7 +273,7 @@ environments {
         vars {
             DB_PVC_SIZE = '50Gi'
             DOCUMENT_PVC_SIZE = '50Gi'
```
Comment (Jsyro) [inline pipeline/config-prod.groovy]: should we take this change to increase document storage pvc? how much room do we have left right now.

File: openshift/_python36.dc.json [hunk 2]
```diff
@@ -192,14 +192,14 @@
...
+        "name": "mds-document-block-storage${SUFFIX}",
         "annotations": {
-          "volume.beta.kubernetes.io/storage-class": "gluster-file",
+          "volume.beta.kubernetes.io/storage-class": "gluster-block",
           "volume.beta.kubernetes.io/storage-provisioner": "kubernetes.io/glusterfs"
         }
       },
       "spec": {
-        "accessModes": ["ReadWriteMany"],
+        "accessModes": ["ReadWriteOnce"],
```
Comment (Jsyro) [inline openshift/_python36.dc.json]: we have more than one backend pod. I don't think we can do this for the backend (we also loose rolling deploys.

File: openshift/_nginx.dc.json [hunk 3]
```diff
@@ -88,14 +88,14 @@
...
+        "name": "mds-logs-block${SUFFIX}",
         "annotations": {
-          "volume.beta.kubernetes.io/storage-class": "gluster-file",
+          "volume.beta.kubernetes.io/storage-class": "gluster-block",
           "volume.beta.kubernetes.io/storage-provisioner": "kubernetes.io/glusterfs"
         }
       },
       "spec": {
-        "accessModes": ["ReadWriteMany"],
+        "accessModes": ["ReadWriteOnce"],
```
Comment (Jsyro) [inline openshift/_nginx.dc.json]: same here, we have nginx on high availability. i think this needs to be RWX.

File: pipeline/config-prod.groovy (hunk 1 above)
Comment (itchison) [reply pipeline/config-prod.groovy]: We have used 5.5GB of the 50GB. Not a pressing concern but more of a mid-term issue.

**Pull request closed without merging**